app/domain/errors.rb

```diff
@@ -270,20 +270,19 @@ module AuthnOidc
     module AuthnIam
 
       InvalidAWSHeaders = ::Util::TrackableErrorClass.new(
-        msg: "Invalid or expired AWS headers: {0}",
+        msg: "Invalid or expired AWS headers: {0-message}",
         code: "CONJ00018E"
       )
 
-      VerificationError = ::Util::TrackableLogMessageClass.new(
+      VerificationError = ::Util::TrackableErrorClass.new(
         msg: "Verification of IAM identity failed with exception: {0-exception}",
         code: "CONJ00063E"
       )
 
-      IdentityVerificationErrorCode = ::Util::TrackableLogMessageClass.new(
-        msg: "Verification of IAM identity failed with HTTP code: {0-http-code}",
-        code: "CONJ00064E"
+      RegionMismatch = ::Util::TrackableErrorClass.new(
```

Comment:
This doesn't appear to be getting used anywhere?

Reply:
Good catch.  I'll remove.